package.json

```diff
@@ -9,7 +9,7 @@
     "start": "next start",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
-    "lint": "eslint . --ext=tsx,ts",
+    "lint": "eslint . --ext=tsx,ts,json --fix",
```

Comment:
Do we need an eslint json plugin to support linting against `.json` files?

Reply:
removed this